pysat/_instrument.py

```diff
@@ -409,11 +409,18 @@ def __init__(self, platform=None, name=None, tag='', inst_id='',
         # Check to make sure value is reasonable
         if self.file_format is not None:
             # Check if it is an iterable string
-            if(not isinstance(self.file_format, str)
-               or (self.file_format.find("{") < 0)
-               or (self.file_format.find("}") < 0)):
+            if isinstance(self.file_format, str):
+                if any([self.file_format.find("{") < 0,
+                        self.file_format.find("}") < 0]):
+                    raise ValueError(''.join(['file format set to default, ',
```

Comment:
This will also trigger if the supplied string is not the default, but doesn't have any '{}' specifiers.

Reply:
That is the intention, unless I misunderstand

Reply:
Yes, it is.  But the message doesn't tell the user that's the problem.  It tells the user it's a different problem, the "default" is now in a separate error (the `else` below).

Reply:
The if/else here is for string input or non-string input, not non-default.  Though since it's an error message, it can't set back to default, so this is confusing